contrib/prometheus/recording-rules-resource.yml

```
@@ -0,0 +1,17 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: host-metering-rules-template
+objects:
+- apiVersion: monitoring.coreos.com/v1
+  kind: PrometheusRule
+  metadata:
+    labels:
+      tenant: rhobs
```

This should reflect the tenant name
```suggestion
      tenant: rhel
```